### ISSUE TITLE
Incorporate SpyBot Immunization List?

### ISSUE THREAD

**Issue by yllekz** [StevenBlack/hosts#1212]
Spybot - Search and Destroy for Windows has a feature called [Immunization](https://www.safer-networking.org/faq/what-is-immunization/) where it adds large amounts of entries to your local Windows hosts file for protection. I was wondering if this could be incorporated into the StevenBlack hosts collection?

**Comment by StevenBlack** [comment]
@yllekz this list is where exactly?

**Comment by yllekz** [comment]
The list is pulled from the Spybot Search and Destroy application on Windows. When the latest update definitions are downloaded, the user runs the Immunization module to pull the latest entries which are written to hosts.

I cannot seem to find any other way to pull the list other than immunizing a system and then scarfing the entries written to the hosts file. The latest Immunization file pulled contains 15638 listings. As an example, I have dumped the one I just pulled to [PasteBin](https://pastebin.com/Mgsq54Wt).

Hope this helps.
